test/guider_test_driver.py

```diff
@@ -72,19 +73,19 @@ def find_guide_star(path, iteration, subframe=None):
 def guide_test_func(config):
 
     star = find_guide_star(
-        r'H:\Observatory Files\Observing Sessions\2020_Data\20200811\TOI_2038-01_40s_R-0001.fits', 1
+        r'H:\Observatory Files\Observing Sessions\2020_Data\20200819\TOI_1531-01_15.000s_R-0001.fits', 1
```

Comment:
In general I'm really not a fan of static file paths.  You should make this a global variable at the very least so if it needs to change it can just be changed in one place.

Reply:
Yes I figured it was fine here since this is just a test driver, but I can still change it if you want.